WalletWasabi.Tests/Helpers/RepeatAttribute.cs

```diff
@@ -0,0 +1,27 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace WalletWasabi.Tests.Helpers
+{
+	public sealed class RepeatAttribute : DataAttribute
```

Comment:
I would link https://stackoverflow.com/a/31878640 here.

Reply:
It is public domain. Adding links does not help devs to understand the code better.

Reply:
The second answer there shows that https://stackoverflow.com/a/55687930 one can probably implement it without doing so many allocations. When xUnit 3.0 will be released and something breaks, this is probably a first place where to look for a fix.